tests/localosmosis/mainnet_state/stateprune.py

```diff
@@ -1,34 +0,0 @@
-import json
```

Comment:
Is this file okay to be removed?
@czarcas7ic

Reply:
Yes!